addons/ondevice-controls/src/types/Boolean.tsx

```diff
@@ -3,40 +3,24 @@ import React from 'react';
 
 export interface BooleanProps {
   onChange: (value: boolean) => void;
-  knob: {
+  arg: {
     name: string;
     value: boolean;
   };
 }
 
-class BooleanType extends React.Component<BooleanProps, {}> {
-  static defaultProps = {
-    knob: {},
-    onChange: (value) => value,
-  };
-
-  static serialize = (value) => (value ? String(value) : null);
-
-  static deserialize = (value) => value === 'true';
-
-  onValueChange = () => {
-    const { onChange, knob } = this.props;
-    onChange(!knob.value);
-  };
-
-  render() {
-    const { knob } = this.props;
-
-    return (
-      <View style={styles.switch}>
-        <Switch testID={knob.name} onValueChange={this.onValueChange} value={knob.value} />
-      </View>
-    );
-  }
-}
+const BooleanType = ({ arg, onChange = (value) => value }: BooleanProps) => (
+  <View style={styles.switch}>
+    <Switch testID={arg.name} onValueChange={() => onChange(!arg.value)} value={arg.value} />
+  </View>
+);
 
 const styles = StyleSheet.create({
   switch: { margin: 10, alignItems: 'flex-start' },
 });
 
+BooleanType.serialize = (value) => (value ? String(value) : null);
```

Comment:
I'm unsure about what these are for and why. Clearly serialization and deserialization, but why do we need them - or do we need them?

Reply:
Honestly I don't know what these are for yet, I think it might have something to do with the storybook/react-native-server package however I need to confirm that. For now I'd prefer to keep them just until I can figure out if they are needed or not.

Reply:
Thanks - let's keep them for now. It would be interesting to know the purpose, so if you spot it (or notice that they aren't necessary) then please ping.

Reply:
Ok will let you know if I figure it out.